swiperefresh/src/main/java/com/google/accompanist/swiperefresh/SwipeRefresh.kt

```diff
@@ -223,12 +287,21 @@ fun SwipeRefresh(
     state: SwipeRefreshState,
     onRefresh: () -> Unit,
     modifier: Modifier = Modifier,
+    position: Position = Position.TOP,
     swipeEnabled: Boolean = true,
     refreshTriggerDistance: Dp = 80.dp,
-    indicatorAlignment: Alignment = Alignment.TopCenter,
+    indicatorHorizontalAlignment: Alignment.Horizontal = Alignment.CenterHorizontally,
```

Comment:
This is an API change we don't really want to do. Please revert the name change.

Thinking about it, can we just keep the old API signature and remove the new `position`? We would need to `check()` that the value doesn't contain a center vertical value, but that seems like the simplest option?

Reply:
If we keep the old API: 
```kotlin
@Composable
fun SwipeRefresh(
    state: SwipeRefreshState,
    onRefresh: () -> Unit,
    modifier: Modifier = Modifier,
    swipeEnabled: Boolean = true,
    refreshTriggerDistance: Dp = 80.dp,
    indicatorAlignment: Alignment = Alignment.TopCenter,
    indicatorPadding: PaddingValues = PaddingValues(0.dp),
    indicator: @Composable (state: SwipeRefreshState, refreshTrigger: Dp) -> Unit = { s, trigger ->
        SwipeRefreshIndicator(s, trigger)
    },
    clipIndicatorToPadding: Boolean = true,
    content: @Composable () -> Unit,
)
```
 How could we know if it's a bottom or top swipe refresh?

Reply:
> How could we know if it's a bottom or top swipe refresh?

Do you need to know? There's only one swipe direction for each `SwipeRefresh()` so you automatically know which it is.

Reply:
When you said old API, did you mean the API before I made this PR:
```kotlin
@Composable
fun SwipeRefresh(
    state: SwipeRefreshState,
    onRefresh: () -> Unit,
    modifier: Modifier = Modifier,
    swipeEnabled: Boolean = true,
    refreshTriggerDistance: Dp = 80.dp,
    indicatorAlignment: Alignment = Alignment.TopCenter,
    indicatorPadding: PaddingValues = PaddingValues(0.dp),
    indicator: @Composable (state: SwipeRefreshState, refreshTrigger: Dp) -> Unit = { s, trigger ->
        SwipeRefreshIndicator(s, trigger)
    },
    clipIndicatorToPadding: Boolean = true,
    content: @Composable () -> Unit,
)
```
or you just want us to keep the old signature for only the `indicatorAlignment ` param?

Please can you respond with the code of signature you were talking about?